### DISCUSSION TITLE
k3os boot has a bug

### DISCUSSION THREAD

**Discussion by jwells38654** [antonym/netboot.xyz#1001] [General] [answered]
Can someone fix the netboot.xyz bootstrap file for TFTP to now append /latest/download/k3os.... as it is no longer the correct GitHub URL path to download the K3os images.  Or how can I fix this?  

FYI- Setting the mirror path still appends the incorrect path mention above.

**Comment by antonym** [comment, marked as answer]
Should be fixed with 46f0182, you can try it directly from the menu today by going to Utilities, choosing netboot.xyz endpoints and development menu.  That should load the changes from the development branch.

**Reply by icepic0** [reply]
the fix works.  I applied the same change to the live boot menu via Pull request #1026
